src/passes/I64ToI32Lowering.cpp

```diff
@@ -1042,7 +1101,7 @@ struct I64ToI32Lowering : public WalkerPass<PostWalker<I64ToI32Lowering>> {
 
   Block* lowerShU(BinaryOp op, Block* result, TempVar&& leftLow,
```

Comment:
How about changing `lowerShU` to `lowerShift`, as now it handles signed as well?